### ISSUE TITLE
ERROR LIMIT/OFFSET is not supported with select_for_update on this database backend.

### ISSUE THREAD

**Issue by Sayeed78618** [pytest-dev/pytest#10340]
Hi
This error comes after a file is processed, beacuse of this error django_q_task table is not updating. In admin panel I can see files to be processed in the Queued tasks but after a file is processed that particular file disappears from queued task not appearing in failed or successful tasks.
Database being used is Oracle 19c.
q_cluster configuation :

`Q_CLUSTER = {
    'name': 'DRPQuee',
    'workers': 2,
    'timeout': 3000,
    'retry': 300000,
    'max_attempts':2,
    'queue_limit': 10000,
    'orm': 'default'
}`

Error log:
Successfully processed file: /home/sayeed/Downloads/BG/BG-0000002330-Product_BG_PDF_0.pdf
15:20:11 [Q] ERROR LIMIT/OFFSET is not supported with select_for_update on this database backend.
15:20:11 [Q] INFO Processed [shade-black-nebraska-jig]

PIP LIST :
`Package                          Version
-------------------------------- --------------------
absl-py                          1.2.0
amqp                             5.1.1
appdirs                          1.4.4
appnope                          0.1.2
apturl                           0.5.2
arrow                            1.2.2
asgiref                          3.3.4
astunparse                       1.6.3
backcall                         0.2.0
bayoo-docx                       0.2.17
bcrypt                           3.1.7
billiard                         3.6.4.0
blessed                          1.19.1
blinker                          1.4
Brlapi                           0.7.0
cachetools                       4.2.2
cchardet                         2.1.7
celery                           5.2.7
certifi                          2020.12.5
chardet                          4.0.0
click                            8.0.1
click-didyoumean                 0.3.0
click-plugins                    1.1.1
click-repl                       0.2.0
colorama                         0.4.3
command-not-found                0.3
coreapi                          2.3.3
coreschema                       0.0.4
cryptography                     2.8
cupshelpers                      1.0
cx-Oracle                        8.3.0
cycler                           0.10.0
dbus-python                      1.2.16
decorator                        4.4.2
defer                            1.0.6
dell-recovery                    0.0.0
distlib                          0.3.1
distro                           1.4.0
distro-info                      0.23ubuntu1
Django                           3.2.2
django-environ                   0.4.5
django-picklefield               3.0.1
django-q                         1.3.9
django-rest-swagger              2.2.0
django-settings-export           1.2.1
djangorestframework              3.12.4
djangorestframework-simplejwt    4.7.1
docopt                           0.6.2
docx                             0.2.4
drf-yasg                         1.20.0
duplicity                        0.8.12.0
entrypoints                      0.3
et-xmlfile                       1.1.0
fasteners                        0.14.1
fastprogress                     1.0.0
filelock                         3.0.12
flatbuffers                      22.9.24
future                           0.18.2
fuzzywuzzy                       0.18.0
gast                             0.4.0
gensim                           4.2.0
google-auth                      1.30.1
google-auth-oauthlib             0.4.4
google-pasta                     0.2.0
greenlet                         1.1.2
grpcio                           1.34.1
h5py                             3.1.0
httplib2                         0.14.0
huggingface-hub                  0.7.0
idna                             2.10
importlib-metadata               4.11.4
inflection                       0.5.1
ipython                          7.23.1
ipython-genutils                 0.2.0
itypes                           1.2.0
jedi                             0.18.0
jieba                            0.42.1
Jinja2                           3.1.2
joblib                           1.0.1
keras                            2.8.0
keras-bert                       0.89.0
keras-embed-sim                  0.10.0
keras-layer-normalization        0.16.0
keras-multi-head                 0.29.0
keras-pos-embd                   0.13.0
keras-position-wise-feed-forward 0.8.0
Keras-Preprocessing              1.1.2
keras-self-attention             0.51.0
keras-transformer                0.40.0
keyring                          18.0.1
kiwisolver                       1.3.1
kombu                            5.2.4
ktrain                           0.31.0
langdetect                       1.0.9
language-selector                0.1
launchpadlib                     1.10.13
lazr.restfulclient               0.14.2
lazr.uri                         1.0.3
libclang                         14.0.1
lockfile                         0.12.2
louis                            3.12.0
lxml                             4.6.3
macaroonbakery                   1.3.1
Mako                             1.1.0
Markdown                         3.3.4
MarkupSafe                       2.1.1
matplotlib                       3.4.2
matplotlib-inline                0.1.2
monotonic                        1.5
netifaces                        0.10.4
networkx                         2.5.1
nltk                             3.7
num2words                        0.5.10
numpy                            1.22.4
oauthlib                         3.1.0
olefile                          0.46
openapi-codec                    1.3.2
openpyxl                         3.0.10
opt-einsum                       3.3.0
packaging                        20.9
pandas                           1.2.4
paramiko                         2.6.0
parso                            0.8.2
pexpect                          4.8.0
pickleshare                      0.7.5
Pillow                           8.2.0
pip                              22.2.2
progressbar                      2.5
prompt-toolkit                   3.0.18
protobuf                         3.17.1
ptyprocess                       0.7.0
pyasn1                           0.4.8
pyasn1-modules                   0.2.8
pycairo                          1.16.2
pycups                           1.9.73
Pygments                         2.9.0
PyGObject                        3.36.0
PyJWT                            2.1.0
pymacaroons                      0.13.0
PyNaCl                           1.3.0
pyp3rclip                        1.8.0
pyparsing                        2.4.7
pyperclip                        1.8.2
pyRFC3339                        1.1
pytesseract                      0.3.9
python-apt                       2.0.0+ubuntu0.20.4.7
python-dateutil                  2.8.1
python-debian                    0.1.36ubuntu1
python-docx                      0.8.11
pytz                             2021.1
pyxdg                            0.26
PyYAML                           5.4.1
redis                            3.5.3
regex                            2021.4.4
reportlab                        3.5.34
requests                         2.25.1
requests-oauthlib                1.3.0
requests-toolbelt                0.9.1
requests-unixsocket              0.2.0
rsa                              4.7.2
ruamel.yaml                      0.17.21
ruamel.yaml.clib                 0.2.6
sacremoses                       0.0.45
scikit-learn                     0.23.2
scipy                            1.6.3
SecretStorage                    2.3.1
sentencepiece                    0.1.95
seqeval                          0.0.19
setuptools                       45.2.0
shortuuid                        1.0.9
simplejson                       3.16.0
six                              1.15.0
smart-open                       6.0.0
sos                              4.3
SQLAlchemy                       1.4.37
sqlparse                         0.4.1
syntok                           1.3.3
systemd-python                   234
templates                        0.0.5
tensorboard                      2.8.0
tensorboard-data-server          0.6.1
tensorboard-plugin-wit           1.8.0
tensorflow                       2.8.0
tensorflow-estimator             2.10.0
tensorflow-io-gcs-filesystem     0.26.0
termcolor                        1.1.0
tf-estimator-nightly             2.8.0.dev2021122109
threadpoolctl                    2.1.0
tika                             1.24
tokenizers                       0.10.3
tqdm                             4.61.0
traitlets                        5.0.5
transformers                     4.10.3
typing-extensions                3.7.4.3
ubuntu-advantage-tools           27.8
ubuntu-drivers-common            0.0.0
ufw                              0.36
unattended-upgrades              0.1
uritemplate                      4.1.1
urllib3                          1.26.4
usb-creator                      0.3.7
vine                             5.0.0
virtualenv                       20.4.4
wadllib                          1.3.3
wcwidth                          0.2.5
Werkzeug                         2.0.1
wheel                            0.37.1
Whoosh                           2.7.4
wrapt                            1.12.1
xkit                             0.0.0
xlrd                             2.0.1
zipp                             3.8.0
`

Regards

**Comment by Sayeed78618** [comment]
Iam Sorry by mistake I reported to wrong branch
